src/__integrations__/pagopaApi.checkStatus.integration.test.ts

```diff
@@ -0,0 +1,137 @@
+import { Server } from 'http';
+
+import { Millisecond } from 'italia-ts-commons/lib/units';
+import nodeFetch from 'node-fetch';
+// Needed to patch the globals
+import 'abort-controller/polyfill';
+import { createHttpTerminator } from 'http-terminator';
+import { retryingFetch } from '../utils/fetch';
+
+import { createClient } from '../../generated/definitions/pagopa/client';
+
+import pm from './pm';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any,functional/immutable-data
+(global as any).fetch = nodeFetch;
+
+// Client for the PagoPA PaymentManager
+describe('Payment Manager Client', () => {
+  const HOST = process.env.PAYMENT_MANAGER_STUB_HOST as string;
+  const PORT = process.env.PAYMENT_MANAGER_STUB_PORT ? parseInt(process.env.PAYMENT_MANAGER_STUB_PORT, 10) : 5000;
+
+  // eslint-disable-next-line functional/no-let
+  let pmMockServer: Server;
+
+  // eslint-disable-next-line functional/no-let
+  let paymentManagerClient;
+
+  beforeAll(() => {
+    pmMockServer = pm.listen(PORT, HOST);
+
+    paymentManagerClient = createClient({
+      baseUrl: `http://${HOST}:${PORT}`,
+      fetchApi: retryingFetch(fetch, 5000 as Millisecond, 5),
```

Comment:
@infantesimone Do you have planned to add the polling when we'll integrate the call to `checkStatus` in FE?

Reply:
Yes, but in another pr because I don't know if we can use constantPollingFetch defined in fetch.ts